test/common/testvm.py

```diff
@@ -77,7 +77,7 @@ def handle_timeout(self, signum, frame):
                 self.machine.ssh_process.terminate()
             self.machine.disconnect()
 
-        raise Exception(self.error_message)
+        raise RuntimeError(self.error_message)
```

Comment:
Right, `Exception` is the mother of all evilW^W^W^W^base class of all exceptions ☺